.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+Resolves #
+
+### Standard stuff
+
+- [x] The PR title is of form 'issue_title #issue_number'
+- [x] If no issue is referenced, my changes are simple and summarized in title - and in max three sentences in the description.  
+      If not please make an issue - so the need for change is motivated and validated.
+- [x] I'll be patient as I understand responses can take days.
+- [x] I have formatted with `cargo fmt`.
```

Comment:
the actions scripts use `cargo fmt --all -- --check`, so maybe change this to cargo fmt --all

on that note, maybe also add the other required quality gate `cargo clippy --release --all-targets -- -D warnings`

Reply:
@kobe-reygel Good point, added your suggestions.